package.json

```diff
@@ -8,16 +8,38 @@
     "start": "next start -p $PORT"
   },
   "dependencies": {
-    "@emotion/react": "11.4.1",
-    "@emotion/styled": "11.3.0",
-    "@mui/icons-material": "5.0.0-rc.0",
-    "@mui/material": "5.0.0-rc.0",
+    "@emotion/react": "11.7.0",
+    "@emotion/styled": "11.6.0",
+    "@gooddollar/goodprotocol": "1.0.0",
+    "@mui/icons-material": "5.2.0",
+    "@mui/material": "5.2.2",
+    "@walletconnect/web3-provider": "^1.6.6",
+    "assert": "^2.0.0",
+    "axios": "^0.24.0",
+    "crypto-browserify": "^3.12.0",
+    "https-browserify": "^1.0.0",
     "merkle-tree-solidity": "1.0.8",
-    "next": "latest",
+    "next": "12.0.4",
+    "os-browserify": "^0.3.0",
     "react": "17.0.2",
-    "react-dom": "17.0.2"
+    "react-dom": "17.0.2",
+    "stream-browserify": "^3.0.0",
+    "stream-http": "^3.2.0",
```

Comment:
stream/crypto/browserify are they all needed?

Reply:
no, not at all actually. can't remember why I added them